src/inverseRobot.js

```diff
@@ -25,6 +25,19 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const robot1 = {};
+
+  for (let i = 0; i < Object.values(robot).length; i++) {
+    if (Object.values(robot).slice(i + 1).includes(Object.values(robot)[i])) {
```

Comment:
this looks overcomplicated, try to shorten it a bit, you can look in [checklist](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md) 2nd item to find an easier solution